src/DataCollector/GraphQLCollector.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\DataCollector;
+
+use GraphQL\Error\SyntaxError;
+use GraphQL\Language\AST\DocumentNode;
+use GraphQL\Language\AST\FieldNode;
+use GraphQL\Language\AST\OperationDefinitionNode;
+use GraphQL\Language\Parser;
+use Overblog\GraphQLBundle\Event\ExecutorResultEvent;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
```

Comment:
I would suggest to remove all all @return tags in the PHPDoc and use type-hints instead. PHP-CS-Fixer will change this automatically in the future.